### PR TITLE
LRDOCS-5939 refer to liferay themes SDK as Liferay JS Toolkit

### PR DESCRIPTION
**Pull request by mwilliams2014**
@codyhoag here are the Liferay JS Toolkit name changes. I've tagged you in the docs that should affect you. Please review to make sure. Thanks!

### CODE REVIEW

File: develop/tutorials/articles/02-from-liferay-6-to-liferay-7/04-improved-developer-tooling/01-from-the-plugins-sdk-to-workspace.markdown [hunk 1]
```diff
@@ -41,7 +41,8 @@ Here's the Workspace anatomy:
 - `gradle/` &rarr; holds the Gradle wrapper files
 - `modules/` &rarr; holds module projects
 - `plugins-sdk/` (generated) &rarr; holds plugins from previous releases
```
Comment (mwilliams2014) [inline develop/tutorials/articles/02-from-liferay-6-to-liferay-7/04-improved-developer-tooling/01-from-the-plugins-sdk-to-workspace.markdown]: @codyhoag

File: develop/tutorials/articles/100-tooling/02-liferay-ide/06-creating-themes-with-liferay-ide.markdown [hunk 2]
```diff
@@ -36,10 +36,10 @@ files you wish to change and copy them into your theme's `webapp` folder.
 **Important:** Under the hood, Dev Studio is using the
 [theme project template](/develop/reference/-/knowledge_base/7-1/theme-template).
 The WAR-style theme created by Dev Studio is Gradle/Maven based; this differs
-from the
-[Theme Generator](/develop/tutorials/-/knowledge_base/7-1/creating-themes) theme
-created with the
-[Liferay Themes Node.js SDK](https://github.com/liferay/liferay-themes-sdk/tree/master/packages).
+from themes created with the
```
Comment (mwilliams2014) [inline develop/tutorials/articles/100-tooling/02-liferay-ide/06-creating-themes-with-liferay-ide.markdown]: @codyhoag

File: develop/tutorials/articles/100-tooling/04-liferay-workspace/02-configuring-a-liferay-workspace.markdown [hunk 3]
```diff
@@ -23,8 +23,8 @@ The top-level files/folder of a Liferay (Gradle) Workspace are outlined below:
   [Using a Plugins SDK from Your Workspace](/develop/tutorials/-/knowledge_base/7-0/configuring-a-liferay-workspace#using-a-plugins-sdk-from-your-workspace)
   section for more information. The Plugins SDK is no longer available for
   @product-ver@.
```
Comment (mwilliams2014) [inline develop/tutorials/articles/100-tooling/04-liferay-workspace/02-configuring-a-liferay-workspace.markdown]: @codyhoag

File: develop/tutorials/articles/100-tooling/04-liferay-workspace/06-managing-themes-in-liferay-workspace.markdown [hunk 4]
```diff
@@ -2,34 +2,35 @@
 
 Creating a @product@ theme can be accomplished using two different tools:
 
-- [Liferay Themes SDK](/develop/tutorials/-/knowledge_base/7-1/creating-themes)
-  (Node.js-based)
+- [Liferay Theme Generator](/develop/tutorials/-/knowledge_base/7-1/creating-themes)
```
Comment (mwilliams2014) [inline develop/tutorials/articles/100-tooling/04-liferay-workspace/06-managing-themes-in-liferay-workspace.markdown]: @codyhoag

File: develop/tutorials/code/com-liferay-docs-guestbook/gradle.properties [hunk 5]
```diff
@@ -115,8 +115,8 @@
     #liferay.workspace.target.platform.version=7.0-GA7
 
     #
-    # Set the folder that contains Node.js-style theme projects. The default
-    # value is "themes".
+    # Set the folder that contains generated theme projects created with the
```
Comment (mwilliams2014) [inline develop/tutorials/code/com-liferay-docs-guestbook/gradle.properties]: @codyhoag I just wanted to get your opinion on this. The `gradle.properties` file is auto-generated, correct? So we would need to change the template for this in the build tools to reflect the name change, right? Do you know how to do that?

**Comment by codyhoag** [comment]
@mwilliams2014 I'm talking with the Dev Tools team about this. I'll ping you tomorrow.

**Pull request closed without merging**